Makefile

```diff
@@ -69,3 +69,12 @@ kustomize_build:
 	# It will fail once we have more than one subdirectory, which will prevent us from
 	# adding untested configurations (this target will have to chage when that happens).
 	./kustomize build prometheus/* 1>/dev/null
+
+.PHONY: install_shellcheck
+install_shellcheck:
+	yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+	yum install -y ShellCheck
+
+.PHONY: lint_shellscript
+lint_shellscript:
+	find . -type f -name "*.sh" -exec shellcheck {} + && echo "lint_shellcheck: SUCCESS"
```

Comment:
Where is the test it's running against?

Reply:
Its linting all shell scripts in a directory and its subdirectories using this command @srivickynesh

Reply:
There are no shell scripts all are converted to makefile.